### ISSUE TITLE
Improve AI.

### ISSUE THREAD

**Issue by MikillRosen** [KOYK/Tiberian-Dawn-Original#144]
I defeated an Impossible AI easily. Here's my replay: [OpenRA-2021-06-01T102211Z.zip](https://github.com/KOYK/Tiberian-Dawn-Original/files/6575544/OpenRA-2021-06-01T102211Z.zip)

**Comment by KOYK** [comment]
Yeah, the current AI lacks early aggression in order to favor a better economy, for later game. 
I will try to fix this for the hard AI levels.
